Source/WebKit/NetworkProcess/cocoa/NetworkProcessCocoa.mm

```diff
@@ -111,6 +132,16 @@ static void initializeNetworkSettings()
     if (auditToken && [NEFilterSource respondsToSelector:@selector(setDelegation:)])
         [NEFilterSource setDelegation:&auditToken.value()];
 #endif
+
+#if PLATFORM(IOS_FAMILY_SIMULATOR)
+    // See TestController::cocoaPlatformInitialize for supporting a local DNS resolver on Mac.
+    static nw_resolver_config_t resolverConfig = nw_resolver_config_create();
```

Comment:
Does this need to be static? Should only be called once by process. Or do we need to keep the config alive for the lifetime of the process?